src/notation/internal/notationinteraction.h

```diff
@@ -248,6 +248,7 @@ class NotationInteraction : public INotationInteraction, public async::Asyncable
         Ms::EditData ed;
         std::vector<Element*> elements;
         std::vector<std::unique_ptr<Ms::ElementGroup> > dragGroups;
+        DragMode mode { DragMode::BothXY };
```

Comment:
We usually use initialization with `=` everywhere. It seems to be easier to read :) But it doesn't matter, initialization with `{}` is more universal.

Reply:
The first file I checked was element.h  which uses brace initialization for everything...apparently the only difference is that assignment initialization can cause auto-conversions (e.g. -1 into an unsigned int), whereas for { } / brace initialization that's an error...

Reply:
> The first file I checked was element.h which uses brace initialization for everything...apparently the only difference is that assignment initialization can cause auto-conversions (e.g. -1 into an unsigned int), whereas for { } / brace initialization that's an error...

Unfortunately, there are a lot of different styles of code in the `libmscore` dir now, because this is code from the old version.
Therefore, it is better to look elsewhere, with new code :)
Also we have code style documentation https://github.com/musescore/MuseScore/wiki/CodeGuidelines